www/index.html

```diff
@@ -466,6 +466,8 @@ <h3>Display settings</h3>
                                         <b>Select app theme</b> (content inversion is experimental):
                                     </label>
                                     <select class="form-control" id="appThemeSelect">
+                                        <option class="auto" value="auto_invert">Auto (generic)</option>
+                                        <option class="auto" value="auto_mwInvert">Auto (Wikimedia)*</option>
                                         <option value="light">Light</option>
```

Comment:
I know it's what @Jaifroid asked for, but I find misleading that the "Light" option (which is the default) is not the first in the list.

Reply:
I just asked for Light to be the default, I think. I agree it can go first in the list (for that very reason). As long as it is the default value when the user starts the app for the first time.

Reply:
I can tell @mossroy has better memory. Anyways,
@mossroy @Jaifroid then where you want to put the `auto` options : just after `light` option or in the end

Reply:
> I can tell @mossroy has better memory

Confirmed! 😉

Reply:
Hahaa XD

Reply:
>  @Jaifroid please let me know where you want to put the `auto` options : just after `light` option or in the end
What about this one

Reply:
> I think this is what @mossroy was implying:
> 
> 1. Light
> 2. Auto (app only)
> 3. Dark (app only)
> 2. Auto (generic invert)
> 3. Dark (generic invert)
> 4. Auto (wikimedia invert)*
> 5. Dark (wikimedia invert)*
> 
> However, @mossroy I do wonder if it's not a bit overwhelming and we could simplify, at least not have the `Auto (app only)` option?

I should have reloaded my tab before asking 👀
Yeah it seems to be bit scattered cuz the users will just want the auto light or dark to applied and those options we'll provide other options.So ,this is what i think should be implemented 
1.Light
2.Dark(app only) 
3. Dark(generic invert)
4. Dark (wikimedia invert)*
5. Auto (app only)
6. Auto (generic invert)
7. Auto (wikimedia invert)*

I wonder what's mossroy has on his mind

Reply:
What I had in mind was the last proposal above, from @Maneet-k 
However, I also find that it's a lot of options, it becomes confusing.

I was about to suggest to remove the "app only" option (dark or auto), but I'm not sure. I currently find it useless but what would happen if we have a ZIM file that handles dark mode itself (I don't know if we already have some): depending on how it is handled by the ZIM content, applying an invert stylesheet on a content that is already inverted could revert it to light mode...

We should find a way to have good default values for the user. Maybe it implies to have several form fields instead of only one.
For example, keep only Light/Dark/Auto options in this drop-down list, and have another drop-down list appearing when Dark or Auto is selected. This second drop-down list would have Generic invert/App only/Wikimedia invert options, with "Generic invert" selected by default (if we agree that it's the best default option)

Reply:
> @Jaifroid what do you say?

I like the two dropdown options, but I think it's more complex and should maybe be a separate PR after we've nailed this one in terms of the options we want.. Please follow mossroy's advice about the options using the current schema, and then later we can find a way to simplify if necessary. As long as we don't disrupt whatever settings users have currently chosen, I don't think it matters too much if we introduce more options now, and then simplify the UI for selecting them as a separate PR for the following release. Most users will probably just stick with light, or choose one of the two dark options. Hard-core fans of dark/light mode switching will understand the other options.

Reply:
I agree. It can be done in two steps